server/controller/comments.js

```diff
@@ -1,25 +1,34 @@
 /* eslint-disable eol-last */
 // import CommentsModel from '../models/comments';
 import QuestionModel from '../models/question';
+import CommentsModel from '../models/comments';
+import {
+  errorResponse,
+  successResponse,
+} from '../utilities/responseformat';
+
 
 const commentsController = {
 
   async createComment(req, res) {
-    const comment = req.body;
-    const checkquestion = await QuestionModel.getcommentById(comment.commentId);
-    if (!checkquestion) {
-      res.status(404).json({
-        status: 404,
-        error: 'comment does not exist',
-      });
+    const newComment = req.body;
 
-      comment.title = checkquestion.title;
-      comment.body = checkquestion.body;
-      comment.userid = req.user.id;
+    const checkquestion = await QuestionModel.getQuestionById(newComment.questionId);
 
-      const newComment = new CommentsModel(comment);
-      const createdComment = await CommentsModel.createComment(newComment);
+    if (!checkquestion) {
+      return errorResponse(res, 404, 'Question does not exist');
     }
+
+    newComment.title = checkquestion.title;
+    newComment.body = checkquestion.body;
```

Comment:
Why are you duplicating question title and body on the comment? I think the only thing the comment needs from the question is the Id.

Reply:
I thought as much!, it's in the project rubric and I try to follow it as much as possible